udata_hydra/routes/checks.py

```diff
@@ -22,3 +28,26 @@ async def get_all_checks(request: web.Request) -> web.Response:
     if not data:
         raise web.HTTPNotFound()
     return web.json_response([CheckSchema().dump(dict(r)) for r in data])
+
+
+async def create_check(request: web.Request) -> web.Response:
+    """Create a new check"""
+    try:
+        payload: dict = await request.json()
+        url = payload["url"]
+        resource_id = payload["resource_id"]
+    except ValidationError as err:
+        raise web.HTTPBadRequest(text=json.dumps(err.messages))
+    except KeyError as e:
+        raise web.HTTPBadRequest(text=f"Missing key: {e}")
+
+    else:
+        context.monitor().set_status(f'Crawling url "{url}"...')
+
+        async with aiohttp.ClientSession(
+            timeout=None, headers={"user-agent": config.USER_AGENT}
+        ) as session:
+            result = await check_url(url=url, resource_id=resource_id, session=session)
```

Comment:
Wondering if in this specific case we'd like the check to have high priority. Currently `check_url` [always sets priority as low](https://github.com/datagouv/hydra/blob/46b73144792392557255a72f3ee63ed88bf2045f/udata_hydra/crawl.py#L288C27-L288C43). Thoughts?